packs/spore-war-bestiary/book-2-the-secret-of-deathstalk-tower/sporescout.json

```
@@ -713,7 +713,7 @@
         },
         {
             "_id": "7u8sJ9qftjWaiy32",
-            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "img": "systems/pf2e/icons/actions/TwoActions.webp",
```

```suggestion
            "img": "systems/pf2e/icons/default-icons/action.svg",
```
We don't need to bother setting icons on NPC abilities when they're on the actor.